File: Pemrograman1/Modul-3/Soal 4/PRAK304-2410817120019-NadyaRamadhani.py
nilai = int(input())

if nilai == 0:
    print("nol")
elif nilai > 0 and nilai < 10:
    print("satuan")
elif nilai > 10 and nilai < 20:
    print("belasan")
elif nilai == 10 or nilai >= 20 and nilai < 100:
    print("puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Pemrograman1/Modul-3/Soal 4/PRAK304-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
    int nilai;
    printf("");
    scanf("%d", &nilai);

   if(nilai == 0){
        printf("nol");
    }else if(nilai > 0 && nilai < 10){
        printf("satuan");
    }else if(nilai > 10 && nilai < 20){
        printf("belasan");
    }else if(nilai == 10 || nilai >= 20 && nilai < 100){
        printf("puluhan");
    }else{
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}